### PR TITLE
Sumo 1.0.1

### PR DESCRIPTION
**Pull request by eugenevinitsky**
Upgrade for sumo 1.0.1 and color scheme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,19 @@ instead of `yarn webpack` and `python sumo_web3d/sumo_web3d.py`.
 
 ![Screenshot of an elevated highway with buildings](screenshots/elevated-highway.png)
 
+### Run to visualize Flow
```
Comment (danvk) [inline README.md:99]: Did you mean to include this in your PR? It doesn't look relevant to sumo-web3d.

Comment (eugenevinitsky) [reply README.md:99]: Sorry, this PR meant to point at my repo. My bad!
